### PR TITLE
EVA-904: Add description of accessioning architecture to FAQ

### PR DESCRIPTION
**Pull request by kandaj**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -665,34 +665,102 @@ <h3 class="padding-bottom-large">FAQs</h3>
...
                                             <a href="#collapse18" class="accordion-title"><h5>What accessions are administered by the EVA?</h5></a>
                                             <div class="accordion-content" data-tab-content>
-                                                The EVA follows the SRA model for accessioning:
+                                                <p>
+                                                    The EVA follows the SRA model for accessioning:
                                                 <ul>
                                                     <ul>
-                                                    <li>A submitted project is administered a ‘project’ accession. These begin “PRJEB” followed by a numerical sequence.</li>
-                                                    <li>Each analysis object within a project is administered an ‘analysis’ accession. These begin “ERZ” followed by a numerical sequence.</li>
+                                                        <li>A submitted project is administered a ‘project’ accession. These begin “PRJEB” followed by a numerical sequence.</li>
```
Comment (cyenyxe) [inline src/index.html]: Please check the quotes used in this text. Maybe it is a result of copying them from Google Docs, but neither the single or double ones are not ASCII characters and could cause encoding problems, see:

`’` versus `'`
`“` versus `"`

File: src/index.html [hunk 2]
```diff
@@ -665,34 +665,102 @@ <h3 class="padding-bottom-large">FAQs</h3>
...
                                                <p>
-                                                Yes. The EVA is committed to the continuation of existing dbSNP ‘submitted SNP’ (ss) and ‘reference SNP’ (rs) accessions. These accessions shall be retrievable via the
-                                                <a href="?Variant%20Browser&species=btaurus_umd31&selectFilter=region&studies=PRJEB6119%2CPRJEB7061%2CPRJEB14878&region=1%3A3000000-3100000">EVA variant browser</a> and
-                                                <a href="?API">web services API</a>
+                                                   Yes. The EVA is committed to the continuation of existing dbSNP ‘submitted SNP’ (ss) and ‘reference SNP’ (rs) accessions.
+                                               </p>
+                                               <p> The accessions of those dbSNP variants that satisfy the EVA submission requirements will be retrievable via the
+                                                    <a href="?Variant%20Browser&species=hsapiens_grch37&selectFilter=snp&snp=rs55880202">EVA variant browser</a> and
```
Comment (cyenyxe) [inline src/index.html]: This example was probably not very good (my fault), as we should prefer non-human species here. Please replace it with

```
?Variant%20Browser&species=mmusculus_grcm38&selectFilter=snp&snp=rs585444580
```

File: src/index.html [hunk 3]
```diff
@@ -665,34 +665,102 @@ <h3 class="padding-bottom-large">FAQs</h3>
...
+                                            <div class="accordion-content" data-tab-content>
+                                                <p>
+                                                    We will gradually import the dbSNP data, and your species of interest may not have been yet processed. During October 2017, we will add to the website a status report on the transition process.
+                                                </p>
+                                                <p>
+                                                    If a species has been already imported into the EVA, the most probable reason is that the dbSNP variant did not satisfy the EVA submission requirements. Please click <a href="?Submit%20Data">here</a> to learn more about the EVA submission requirements.
+                                                </p>
+                                                <p>
+                                                    If you know the variant satisfied all of them, please communicate the issue via <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a>
```
Comment (cyenyxe) [inline src/index.html]: Let's rephrase this a bit: `If you know a variant satisfied all of them and it is not displayed in our browser, please`...

**Comment by kandaj** [comment]
updated

File: src/index.html [hunk 4]
```diff
@@ -389,13 +389,13 @@ <h3 class="padding-bottom-large">Clinically relevant genetic variation data</h3>
                                             <h5>Annotation</h5>
                                             <p>
                                                 Once variants have been normalised, the EVA uses the <a href="http://www.ensembl.org/info/docs/tools/vep/" target="_blank">Variant Effect Predictor (VEP) </a>  of Ensembl to annotate variant consequences.
-                                                The variant consequences are described using Sequence Ontology terms and both the VEP version and Ensembl gene build used are described via the “i” help bubbles on the <a href="http://www.ebi.ac.uk/eva/?Variant%20Browser" target="_blank">EVA Variant Browser</a>.
+                                                The variant consequences are described using Sequence Ontology terms and both the VEP version and Ensembl gene build used are described via the "i"help bubbles on the <a href="http://www.ebi.ac.uk/eva/?Variant%20Browser" target="_blank">EVA Variant Browser</a>.
```
Comment (cyenyxe) [inline src/index.html]: The whitespace after "i" is missing now.

File: src/index.html [hunk 5]
```diff
@@ -411,7 +411,7 @@ <h3 class="padding-bottom-large">Clinically relevant genetic variation data</h3>
                                             </p>
                                             <p><span style="font-size:14px;font-weight:bold;">Population / sample cohort allele frequency values:</span></p>
                                             <p>
-                                                The EVA accepts submission of <a href="http://pngu.mgh.harvard.edu/~purcell/plink/data.shtml#ped" target="_blank">pedigree </a> files, or structured samples (using “derived_from” and/or “subject” layers), to define populations and cohorts within studies.
+                                                The EVA accepts submission of <a href="http://pngu.mgh.harvard.edu/~purcell/plink/data.shtml#ped" target="_blank">pedigree </a> files, or structured samples (using "derived_from"and/or "subject"layers), to define populations and cohorts within studies.
```
Comment (cyenyxe) [inline src/index.html]: Whitespaces missing after the double quotes.

File: src/index.html [hunk 6]
```diff
@@ -617,7 +617,7 @@ <h4 style="color:#996a44;">Past Collaborations</h4>
                                     <div class="accordion-content" data-tab-content>
                                         <p>
                                             Yes. Data submitted to the EVA can be held privately for up to one year.
-                                            The date of publication is set by the submitter using the “Hold Date” field of the <a href="files/EVA_Submission_template.V1.0.5.xlsx">EVA metadata template</a>.
+                                            The date of publication is set by the submitter using the "Hold Date"field of the <a href="files/EVA_Submission_template.V1.0.5.xlsx">EVA metadata template</a>.
```
Comment (cyenyxe) [inline src/index.html]: Whitespace missing.

**Comment by kandaj** [comment]
fixed

**Pull request merged**